src/skimage/morphology/gray.py

```
@@ -2,52 +2,14 @@
-        else:
-            cval = np.inf
-    elif mode == "min":
-        mode = "constant"
-        if np.issubdtype(dtype, bool):
-            cval = False
-        elif np.issubdtype(dtype, np.integer):
-            cval = np.iinfo(dtype).min
-        else:
-            cval = -np.inf
-    return mode, cval
+__all__ = ['erosion', 'dilation', 'opening', 'closing', 'white_tophat', 'black_tophat']
```

Ruff (0.14.14)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)